src/statement-list/statement_list_parse_tree.cpp

```diff
@@ -127,3 +105,23 @@ void statement_list_parse_treet::swap(statement_list_parse_treet &other)
   function_blocks.swap(other.function_blocks);
   functions.swap(other.functions);
 }
+
+statement_list_parse_treet::functiont::functiont(
+  const irep_idt &name,
+  const float version,
+  const exprt &return_value)
+  : tia_modulet(name, version), return_value(return_value)
+{
+}
+
+void statement_list_parse_treet::instructiont::add_token(codet &token)
+{
+  tokens.push_back(token);
+}
+
+statement_list_parse_treet::var_declarationt::var_declarationt(
+  const irep_idt &identifier,
```

Comment:
Is this format suggested by `clang-format` with the current repository settings?

Reply:
Yes it is.